# Error Handling

When an error occurs, or exception as we call it, Python will normally stop and generate an error message.
We can handle these errors instead using try and except.

The try block lets you test a block of code for errors.

The except block lets you handle the error.

The else block lets you execute code when there is no error.

The finally block lets you execute code, regardless of the result of the try- and except blocks.

## Further information

- [Python Try Except](https://www.w3schools.com/python/python_try_except.asp)
